specification/eventgrid/Azure.Messaging.EventGrid/main.tsp

```diff
@@ -9,7 +9,7 @@ enum ServiceApiVersions {
 }
 
 @useAuth(
-  ApiKeyAuth<ApiKeyLocation.header, "SharedAccessKey"> | OAuth2Auth<[{
+  ApiKeyAuth<ApiKeyLocation.header, "aeg-sas-key"> | OAuth2Auth<[{
```

Comment:
this should be SharedAccessKey not aeg-sas-key

Reply:
It should be actually what Tim said on Teams:
```
import "@typespec/http";

using TypeSpec.Http;
@service({
  title: "Widget Service",
  version: "1.0.0",
})
@useAuth({type: AuthType.http, scheme: "SharedAccessKey"})
namespace DemoService;


op test(): string;
```